config/flog.yml

```diff
@@ -1,2 +1,2 @@
 ---
-threshold: 27.7
+threshold: 33.8
```

Comment:
> Thats a more than 25% increase ... Ideally Flay / Flog scores grow logarithmic to the amount of complexity.

Yeah I agree. I will note that this PR *should* necessarily be a big jump since the number of ast types is almost doubling:

2.3.0 :005 > (Mutant::AST::Types::ALL - Mutant::AST::Types::REGEXP).size
 => 94
2.3.0 :006 > Mutant::AST::Types::ALL.size
 => 162